scss/_patterns_segmented-control.scss

```diff
@@ -0,0 +1,60 @@
+@mixin vf-p-segmented-control {
+  %vf-button-group-borders {
+    border-bottom: 0;
+    border-left: 0;
+    border-right: 1px solid $color-mid-light;
+    border-top: 0;
+
+    &:last-child {
+      border-right: 0;
+    }
+  }
+
+  .p-segmented-control {
+    @extend %vf-has-round-corners;
+
+    border: 1px solid $color-mid-dark;
+    display: inline-flex;
+    margin-bottom: $spv-outer--small-scaleable;
+
+    // stylelint-disable selector-max-type
+    button,
```

Comment:
We shouldn't style class-less elements inside components, and we should avoid overriding style of other class names `p-button` inside components. This creates situations when elements or components look and behave differently in different places.

This should be something like `p-segmented-control__button` and no other external class name should be needed.